rest_framework/permissions.py

```diff
@@ -44,8 +44,10 @@ class IsAuthenticated(BasePermission):
     """
     Allows access only to authenticated users.
     """
-
+    # DRF should authorize all OPTIONS requests by default #5616
```

Comment:
The comment here isn't really necessary. It can be reduced to `Authorize all OPTIONS requests`, which is self-described in the code change below.

Reply:
okay will delete unnecessary comments, and will do needed tests

Reply:
done! can you please check it again?